generate_kubernetes_enums.js

```diff
@@ -0,0 +1,145 @@
+const fs = require('fs');
+const _ = require('lodash');
+const download = require('download');
+
+const COMMENT_REGEX = /(\/\*([\s\S]*?)\*\/)|(\/\/(.*)$)/gm;
+
+function parse_protocol_buf(field) {
```

Comment:
Perhaps give an example of the kind of input this function handles?  That might make it easier for the reader to see what things like the regexes and the magic number array indexes are doing.